File: Criptografia.java
import java.util.Scanner;
import java.io.File;

public class Criptografia {
	public static void main(String[] args) {
		try {
			Scanner sc = new Scanner(new File("texto_cifrado.txt"));
	
			String cifra = sc.nextLine();
			
			String textoPronto = "";
			
			for (int i = 0; i < cifra.length(); i++) {
				if (cifra.charAt(i) == '4') {
					textoPronto += 'a';
				} else if (cifra.charAt(i) == '3') {
					textoPronto += 'e';
				} else if (cifra.charAt(i) == '1') {
					textoPronto += 'i';
				}  else if (cifra.charAt(i) == '0') {
					textoPronto += 'o';
				} else if (cifra.charAt(i) == '7') {
					textoPronto += 'u';
				} else {
					textoPronto += cifra.charAt(i);
				}
			}
			System.out.println("Texto cifrado: " + cifra);
			System.out.println("Texto pronto: " + textoPronto);
		} catch (Exception e) {
			e.printStackTrace();
		}
	}
}


//4 = a
//3 = e
//1 = i
//0 = o
//7 = u

File: Criptografia.js
const fs = require('fs')

fs.readFile('./texto_cifrado.txt', 'utf8' , (err, data) => {
    if (err) {
        console.error(err)
        return
    }
    var textoPronto = "";
			
    for (var i = 0; i < data.length; i++) {
        if (data.charAt(i) == '4') {
            textoPronto += 'a';
        } else if (data.charAt(i) == '3') {
            textoPronto += 'e';
        } else if (data.charAt(i) == '1') {
            textoPronto += 'i';
        }  else if (data.charAt(i) == '0') {
            textoPronto += 'o';
        } else if (data.charAt(i) == '7') {
            textoPronto += 'u';
        } else {
            textoPronto += data.charAt(i);
        }
    }
    console.log("Texto cifrado: " + data)
    console.log("Texto pronto: " + textoPronto)
})
